lib/graphql-anycable.rb

```diff
@@ -11,33 +11,50 @@
 
 module GraphQL
   module AnyCable
-    def self.use(schema, **opts)
-      schema.use(GraphQL::Subscriptions::AnyCableSubscriptions, **opts)
-    end
+    class << self
+      def use(schema, **opts)
+        schema.use(GraphQL::Subscriptions::AnyCableSubscriptions, **opts)
+      end
 
-    def self.stats(**opts)
-      Stats.new(**opts).collect
-    end
+      def stats(**opts)
+        Stats.new(**opts).collect
+      end
+
+      def redis=(connector)
```

Comment:
Should we keep original `redis` accessor method just in case if some users are still using it? Maybe also add a warning.

Something like this:
```suggestion
      def redis
        warn "Usage of `GraphQL::AnyCable.redis` is deprecated. Instead of `GraphQL::AnyCable.redis.whatever` use `GraphQL::AnyCable.with_redis { |redis| redis.whatever }`"
        @redis ||= with_redis { |conn| conn }
      end

      def redis=(connector)
```